rules/aircraft.yaml

```diff
@@ -1,3 +1,98 @@
+shad:
+	Inherits: ^Aircraft
+	Tooltip:
+		Name: Night Hawk
+		Description: Infantry Transport Helicopter\nUndetectable by radar.\n  Strong vs Infantry\n  Weak vs Vehicles, Aircraft
+	Health:
+		HP: 175
+	Armor:
+		Type: Light
+	Armament:
+		Weapon: BlackHawkCannon
+	Voiced:
+		VoiceSet: NighthawkVoice
+	Buildable:
+		Queue: Vehicle
+		BuildPaletteOrder: 20
+		Prerequisites: ~gaweap
+	Valued:
+		Cost: 1000
+	RevealsShroud:
+		Range: 7c0
+		Type: CenterPosition
+	-AppearsOnRadar:
+	AttackFrontal:
+		Voice: Attack
+	Aircraft:
+		CruiseAltitude: 3072
+		LandWhenIdle: true
+		LandableTerrainTypes: Clear, Rough, Road, DirtRoad
+		InitialFacing: 0
+		ROT: 10
+		Speed: 100
+		AltitudeVelocity: 120
+		Voice: Move
+		AirborneUpgrades: airborne
+		CruisingUpgrades: cruising
+		RepairBuildings:
+		RearmBuildings:
```

Comment:
Indeed that is not required anymore since https://github.com/OpenRA/OpenRA/pull/10233. Removed.